src/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Leaderboards</title>
+</head>
+<body>
+  <header>
+    <h1>Leaderboards</h1>
+  </header>
+  <main>
+    <section>
+      <div class="re-set"><h4>Recent Scores</h4>
+        <button type="reset">Refresh</button>
+      </div>
+      <div class="score-table">
+        <ul>
+          <li>Name: 100</li>
+          <li>Name: 20</li>
+          <li>Name: 50</li>
+          <li>Name: 78</li>
+        </ul>
+      </div>
+    </section>
+    <section>
+      <h4>Add Your Score</h4>
+      <form action="">
+        <input type="text" placeholder="Your name" name="name">
+        <input type="text" placeholder="Your score" name="name">
+        <br>
+        <button  class = "bttn" type="submit">Submit</button>
+      </form>
+    </section>
+  </main>
```

Comment:
- [ ] You did a great job on the design but there are a few issues you need to fix to follow the wireframe design.
 - Please adjust the Leaderboard title to the left side of the screen instead in the center.
 - Please wrap the scoreboard with the border, That will look nice for sure 😄

### Your design

![image](https://user-images.githubusercontent.com/83803414/180184247-d7524b77-6618-4e62-b0b3-637b4e01b3cb.png)

### Original design

![image](https://user-images.githubusercontent.com/83803414/180184306-c6e214b8-19b2-4fd8-80ee-9cdfd179475b.png)